src/main/java/in/projecteka/consentmanager/consent/ConsentArtefactRepository.java

```diff
@@ -166,7 +168,10 @@
         for (Row result : results) {
             artefacts.add(getConsentArtefactRepresentation(result));
         }
-        return artefacts;
+        return artefacts.stream().
+                sorted(Comparator.comparing(c ->
+                        c.getConsentDetail().getLastUpdated(), Comparator.reverseOrder()))
+                .collect(Collectors.toList());
```

Comment:
We can sort it in the query itself on basis of date_modified field. If I am not wrong.